invenio_communities/communities/schema.py

```diff
@@ -265,14 +265,25 @@ class CommunitySchema(BaseCommunitySchema):
     """Community schema."""
 
     parent = NestedAttribute(CommunityParentSchema, dump_only=True)
-
+    # When loading the parent community we don't require any fields as we will only use the ID
+    id = fields.String(dump_only=False)
+    metadata = NestedAttribute(CommunityMetadataSchema, required=False) 
+    access = NestedAttribute(CommunityAccessSchema, required=False)
```

Comment:
minor: it's actually ok to pass all the fields even when just updating the parent, so we can remove these. This is so that we keep the `PUT` semantics, which requires the entire representation (we do divert though for `theme` and `parent`, where we have some `PATCH` behavior).

Reply:
Right, sorry this is a leftover, I will remove it